### DISCUSSION TITLE
What is some popular stacks people are using in the industry

### DISCUSSION THREAD

**Discussion by paschalidi** [BretFisher/ama#123] [Q&A]
Hey @BretFisher thanks for all the amazing content you put out there! 

I was curious if you have the time to help me with my devops transformation and if we are lucky this could perhaps sparkle some ideas for future content that people like me would find interesting.

I want to land a job as a devops engineer some time in the near future. I am curious which tools/techs shall I start learning. I have heard your great podcast on the topic with the intern Devops guest of yours which has some great leads. However as you will see there are more interesting questions on the topic.

So shortly about me. I am a frontend dev with six years of experience in web apps and startups. I have worked with docker but I am not a master on it just yet. You can say that, I can simply use it to get stuff done. I also have worked on the aws platform, but you cant say I know it inside out either. But I guess few people do anyways.

My plan is to learn the followings in this order. 
Ansible & Terraform. When I have an understanding of these tools I wanna see if it makes sense to also get some depth in Cloudformation or get the aws devops certificate. Then I wanna go in depth on Docker and then Swarn and perhaps Kubernetes. For those I bought your udemy courses. Last I want to get my hands on Prometheus. 

You got any advice or thoughts on how I could plan this different? Would you say I miss something out or there would be a better stack to learn? To be honest I am not sure if Cloudformation is really needed or even if it would be smarter to just learn Cloudformation and leave Ansible & Terraform out of the equation. 

----
Changing gears a bit and looking things from a distance, I feel at the moment a bit blind when it comes to some popular stacks devops engineers are using in real world projects. That includes tooling. 

For example if you read above for me it isnt really sure what ground cloudformation can cover. Can I use it for instance instead Ansible or Terraform or is it a bad idea. Sometimes I have the feeling aws provides for everything but the industry hasnt adopted all their tools and I wouldnt like to spent a week learning a tool just to realise no one is really using it. Feels like I am in an aws echo-chamber sometimes this is why I would like to understand what are some stacks successful projects are using when it comes to devops and automation.
 
In the same way I am not sure if learning the aws alternative or swarm and kubernetes is a valid decision. I heard your podcast on the topic where you and some guest of yours were discussing this exact topic. 

I hope this makes sense and I hope this would help you get the pulse of what some junior devops might be thinking or asking themselves.

**Comment by BretFisher** [comment]
You can see my opinions on tech and tools to learn in the DevOps AMA: https://github.com/BretFisher/ama/issues/7

There's a website that tracks popular stacks of tools: https://stackshare.io

I hope that helps!

**Comment by paschalidi** [comment]
Thanks a lot for coming back to this! Will check it out @BretFisher
